res/sections/26-Les16052017.tex

```diff
@@ -18,7 +18,220 @@ \subsubsection{Monitoraggio e Valutazione}
 adeguate per preservare la privacy, avrà delle multe fino al 5\% del fatturato
 e la possibilità di andare a processo.
 
-\subsubsection{Standard di sicurezza}
+
+
+
+
+
+\section{CMM}
+
+Il \textit{Capability Maturity Model} (CMM) è nato come modello per
+valutare la maturità della capacità
+di sviluppo del Software. Ora questa categorizzazione viene utilizzata
+anche per valutare la maturità di un'azienda.
+In particolare nel COBIT questo modello trova impiego per valutare la
+capacità di governare i processi di gestione dell'IT.
+I vari livelli del CMM sono riportati nella Figura~\ref{fig:cmm:levels}.
+
+In generale gli investitori saranno più portati a investire in aziende che
+hanno un CMM alto rispetto ad uno basso.
+I livelli bassi, dal livello 2 in giù, sono legati fortemente all'abilità
+dei singoli.
+I livelli alti sono caratterizzati
+dalla documentazione dei processi (Lv. 3),
+dalle metriche (Lv. 4) e dal miglioramento continuo (Lv. 5).
+Inoltre dal Livello 4 in su,
+l'automatizzazione dei processi di controllo permette un risparmio sul
+TCO (Total Cost of Ownership).
+
+
+
+
+
+\begin{figure}[h!]
+        \begin{center}
+                \includegraphics[scale=2.0]{res/img/maturity_level}
+        \end{center}
+        \caption{I 5 livelli del Capability Maturity Model (CMM).}
+        \label{fig:cmm:levels}
+\end{figure}
+
+
+
+I cinque livelli sono:
+\begin{enumerate}
+  \item \textbf{Non-esistente} (0): i processi gestionali sono del tutto
+  assenti.
+  \item \textbf{Initial} (1): si ``vive alla giornata''. Si è estremamente
+  reattivi agli eventi che accadono e non è in atto alcun tipo di
+  pianificazione e i processi sono ad hoc e disorganizzati.
+  \item \textbf{Managed} (2): si ha ancora un approccio di tipo reattivo agli
+  eventi che accadono, ma si comincia ad definire processi ed ad avere
+  pianificazioni per i progetti;
+  \item \textbf{Defined} (3): si cominciano a definire processi anche per
+  l'organizzazione in toto e si comincia ad avere un approccio proattivo ai
+  problemi e si \textbf{documentano} i processi.
+  \item \textbf{Qualitatively Managed} (4): vengono applicate misurazioni e
+  \textbf{metriche} ai processi per poter controllarli e migliorarli continuamente;
+  \item \textbf{Optimized} (5): una volta che è in atto il \textbf{miglioramento continuo}
+  dei processi che vengono a loro volta misurati tramite metriche l'azienda
+  può puntare ad ottimizzarli il più possibile.
+\end{enumerate}
+Il CMM è stato superato dal CMMI \textit{Capability Maturity Model Integration}.
+
+
+
+\subsection{Livello 0}
```

Comment:
Ok e questi livelli si riferiscono a cosa? `CMM` o `COBIT`?

Perché se si riferiscono a CMM non c'è già la lista puntata (qualche linea prima 😅)

Reply:
Allora ripeto: sostanzialmente il COBIT adotta i livelli del CMM per valutare la maturità dei propri "processi di gestione dell'IT" (cit. del manuale del COBIT 4.1). 
Infine si, si ripete tre volte la lista. Sarebbe meglio mergiare..

Reply:
Boh io direi che è una cosa da lasciare ai posteri, l'importante che noi sappiamo che siano la stessa cosa...